src/event-handler.js

```diff
@@ -1,37 +1,93 @@
-var optimizelyEvents = require('./optimizely-defined-events');
+var optimizelyWebXEvents = require('./optimizely-x-defined-events');
+var optimizelyFullStackEvents = require('./optimizely-fs-defined-events');
 
 function EventHandler(common) {
     this.common = common || {};
 }
 
 EventHandler.prototype.logEvent = function(event) {
-    if (optimizelyEvents.events[event.EventName]) {
-        var optimizelyEvent = {
+    if (optimizelyWebXEvents.events[event.EventName]) {
+        var optimizelyWebXEvent = {
             type: 'event',
             eventName: event.EventName
         };
 
         if (event.EventAttributes) {
-            optimizelyEvent.tags = event.EventAttributes;
+            optimizelyWebXEvent.tags = event.EventAttributes;
         }
 
         if (event.CustomFlags && event.CustomFlags['Optimizely.Value']) {
-            optimizelyEvent.tags.value = event.CustomFlags['Optimizely.Value'];
+            optimizelyWebXEvent.tags.value = event.CustomFlags['Optimizely.Value'];
         }
-        window['optimizely'].push(optimizelyEvent);
+        window['optimizely'].push(optimizelyWebXEvent);
+    }
+
+    // if optimizely full stack is being used
+    if (window.optimizelyClientInstance) {
```

Comment:
this is currently letting optimizely track all events.  we should make this 

include optimizelyWebXEvents.events[event.EventName]
```suggestion
    if (window.optimizelyClientInstance && optimizelyFullStackEvents.events[event.EventName]) {
```